src/main/java/book/store/service/impl/ShoppingCartServiceImpl.java

```diff
@@ -0,0 +1,99 @@
+package book.store.service.impl;
+
+import book.store.dto.CartItemRequestDto;
+import book.store.dto.ShoppingCartResponseDto;
+import book.store.mapper.ShoppingCartMapper;
+import book.store.model.Book;
+import book.store.model.CartItem;
+import book.store.model.ShoppingCart;
+import book.store.model.User;
+import book.store.repository.BookRepository;
+import book.store.repository.CartItemRepository;
+import book.store.repository.ShoppingCartRepository;
+import book.store.service.ShoppingCartService;
+import jakarta.persistence.EntityNotFoundException;
+import java.util.Optional;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@RequiredArgsConstructor
+@Service
+public class ShoppingCartServiceImpl implements ShoppingCartService {
+    private final ShoppingCartRepository shoppingCartRepository;
+    private final CartItemRepository cartItemRepository;
+    private final BookRepository bookRepository;
+    private final ShoppingCartMapper shoppingCartMapper;
+
+    @Override
+    public void createShoppingCart(User user) {
+        ShoppingCart shoppingCart = new ShoppingCart();
+        shoppingCart.setUser(user);
+        shoppingCartRepository.save(shoppingCart);
+    }
+
+    @Override
```

Comment:
Should we use Transactional?

Reply:
OK. But i wonder, how should it be inconsistent, if there's only one saving to repository action?

Reply:
hmm..I think there is misunderstanding. 

I texted about consistency related to `@Repository `annotation